### ISSUE TITLE
Some Unity games can't launch if a non-native resolution is used in the libTAS settings

### ISSUE THREAD

**Issue by DemoJameson** [clementgallet/libTAS#634]
Game: https://store.steampowered.com/app/3241750/Ninja_Ming_Demo

```
X Error of failed request:  BadRRCrtc (invalid Crtc parameter)
  Major opcode of failed request:  139 (RANDR)
  Minor opcode of failed request:  27 (RRGetCrtcTransform)
  Crtc id in failed request: 0x1
  Serial number of failed request:  54
  Current serial number in output stream:  54
```

**Comment by DemoJameson** [comment]
The same error reported by others
https://discord.com/channels/726811446498820198/788850800347709491/1275566543866040441

**Comment by DemoJameson** [comment]
Fixed in a0c76ce35f5774712945378c7407477c4b10ffb2
